src/components/Rating.vue

```diff
@@ -13,9 +13,9 @@
               :max="max"
               :count="facet.count"
         >
-          <template v-for="(n, key) in max">
-            <span v-if="n <= facet.value" :class="bem('star')" :key="key">&#9733;</span>
-            <span v-else :class="bem('star', 'empty')" :key="key">&#9734;</span>
+          <template v-for="n in max">
+            <span v-if="n <= facet.value" :class="bem('star')" :key="n">&#9733;</span>
+            <span v-else :class="bem('star', 'empty')" :key="n">&#9734;</span>
```

Comment:
yes given the conclusion in the comments of this PR, the keys here are unique enough.
I need to check though if we need to cast the integer to string.